internal/provider/utility.go

```diff
@@ -212,14 +212,14 @@ func flattenDSourceHooks(hooks []dctapi.Hook, oldList []dctapi.SourceOperation)
 
 func flattenTags(tags []dctapi.Tag) []interface{} {
 	if tags != nil {
-		returnedTags := make([]interface{}, len(tags))
+		result := make([]interface{}, len(tags))
```

Comment:
I think this is just refactoring with no functional usage.

Reply:
Fixed